tensorflow_app/views/export_graphdef.py

```diff
@@ -26,6 +26,10 @@ def export_to_tensorflow(request):
     os.chdir(BASE_DIR + '/tensorflow_app/views/')
     os.system('KERAS_BACKEND=tensorflow python json2pbtxt.py -input_file ' +
               randomId + '.json -output_file ' + randomId)
+    
+    tf.keras.backend.clear_session()
```

Comment:
Same issue as above, try using `keras.backend` instead. And since you have imported `clear_session()`, why not just directly call `clear_session()`? (Instead of `tf.keras.backend.clear_session()`).